utils/dtoverlay/Makefile

```diff
@@ -0,0 +1,37 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=dtoverlay
+PKG_VERSION:=2016-07-06
+PKG_RELEASE=$(PKG_SOURCE_VERSION)
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=git://github.com/ya-mouse/dtoverlay.git
```

Comment:
Please use http URL which is more friendly behind proxy.

It looks like you bundled various tools in you own, personal repo? Usually, OpenWrt/LEDE packages are modelled to use existing/official sources... Don't get me wrong - but I'm not sure, whether this is the best approach...